buildSrc/src/main/groovy/Docker.groovy

```diff
@@ -334,18 +345,14 @@ class Docker {
 
                     remotePath.set(cfg.containerOutPath)
                     outputDir.set(project.file(dockerCopyLocation))
+                    finalizedBy syncAfterBuildAndRun
                 }
             }
-            // Sync outputs to the desired location
-            return project.tasks.register(taskName, Sync) { sync ->
-                sync.with {
-                    dependsOn buildAndRun
-
-                    // run the provided closure first
-                    cfg.copyOut.execute(sync)
 
-                    // then set the from location
-                    from dockerCopyLocation
+            return project.tasks.register(taskName) { task ->
```

Comment:
this doesn't make sense to me, it doesn't do anything, but it still claims it has outputs - why not return syncAfterBuildAndRun, which already has the outputs declared (by being of type Sync)?

Reply:
We discussed over slack and you said you will followup.  The gist of the discussion was:
```
Because syncAfterBuildAndRun does not depend on buildAndRun
so buildAndRun wouldn't run.  We can't just return buildAndRun because
the rest of the code wants both ${task}Run and ${task} to exist.
```
You said you wanted to look at the rest of the code to see if we could remove the need for having both `${task}Run` and `$task` exist.